### ISSUE TITLE
config.js to include server names

### ISSUE THREAD

**Issue by onvrb** [Itokoyamato/TokoVOIP_TS3#124]
This will be so helpful to use in a Docker stack. I've already made a simple Dockerfile to run ws_server on, adding names instead of the IP's (that could change, even though I use static IP's) would be helpful.

**Comment by Itokoyamato** [comment]
I agree some config should be improved to be used more easily in a docker stack. Are you talking about using domain names or environment variables ?

**Comment by onvrb** [comment]
domain names resolved by the container itself (or the network of the docker stack)

basically add "fivem" to the config.js so they resolve to the container named "fivem"

**Comment by Itokoyamato** [comment]
Ah yes right, will see what we can do, but that's definitely not on the priority list for me. Always open to PRs however
